package.json

```diff
@@ -5,15 +5,15 @@
   "description": "ECMAScript Internationalization API Specification",
   "scripts": {
     "build-es2020": "git remote remove origin && git remote add origin \"git@github.com:$TRAVIS_REPO_SLUG.git\" && git fetch --quiet origin && git checkout --quiet es2020 && mkdir \"out/2020\" && cp -R img \"out/2020\" && ecmarkup --verbose spec/index.html out/2020/index.html --css out/2020/ecmarkup.css --js out/2020/ecmarkup.js",
-    "build-master": "mkdir out && cp -R img out && ecmarkup --verbose spec/index.html out/index.html --css out/ecmarkup.css --js out/ecmarkup.js",
+    "build-master": "mkdir -p out && cp -R img out && ecmarkup --lint-spec --verbose spec/index.html out/index.html --css out/ecmarkup.css --js out/ecmarkup.js",
```

Comment:
I had it in a separate commit, but yes it is necessary if you want to run `npm run build-master` twice in a row.

Reply:
I don't have strong feelings about this, but adding `-p` here will suppress the error that would otherwise abort progress when an "out" directory already exists. The [ECMA 262 approach](https://github.com/tc39/ecma262/blob/master/package.json) is probably better, but also too complex for my taste.

Reply:
It was just a fix for convenience, if it has other disadvantages then I'll go ahead and remove it.